PyInstaller/compat.py

```diff
@@ -199,6 +199,27 @@
     modname_tkinter = 'tkinter'
 
 
+if is_win:
+    try:
+        import pywintypes
+        if is_py3:
+            # pywintypes.Unicode() breaks under Python 3, just stub it out if we're running
+            # under such a Python version.
+            pywintypes.Unicode = str
+    except ImportError:
+        # Do nothing here. User should be warned when check_requirements() is
+        # called.
+        pass
+
+def encode(text):
+    """
+    Encodes a string as bytes.
+    """
+    if is_py3:
+        return str.encode(text)
```

Comment:
Encoding not specified. Text will be encoded using `sys.getdefaultencoding()` which is UTF-8 on Py3.5, and ASCII on Py2.7

This function is used to encode text that will be put into VS_VERSION_INFO structs. MSDN documentation says these are "Unicode" strings. On MSDN, "Unicode" always means "UTF-16".